packages/controller-utils/src/util.ts

```diff
@@ -200,10 +200,10 @@ export function toHex(value: number | string | BN): Hex {
  * @param logError - Determines if the error should be logged.
  * @returns Promise resolving to the result of the async operation.
```

Comment:
We should probably add a jsdoc entry for the new generic param.
```suggestion
 * @template T - Type of the result of the async operation
 * @returns Promise resolving to the result of the async operation.
```

Reply:
Done in https://github.com/MetaMask/core/pull/3629/commits/ac1629d42b54b1b76ba66c04321d19e0976c514d